tests/expense.test.js

```diff
@@ -54,7 +54,7 @@ describe('Expense', () => {
         title: 'Buy a new laptop',
         amount: 999,
         category: 'Electronics',
-        note: 'I need a new laptop',
+        note: 'I need a new laptop'
       };
 
       await api.post('/expenses').send(expenseData).expect(400);
```

Comment:
Ensure that the test setup includes creating a user before testing for a non-existent user to avoid false positives.